File: windows/mssql_connection_plugin.cpp
// This is a dummy plugin implementation required by Flutter's plugin system.
// The actual FFI bindings are handled by Dart FFI directly loading the
// bundled FreeTDS DLLs.

#include "include/mssql_connection/mssql_connection_plugin_c_api.h"

#include <flutter/plugin_registrar_windows.h>

void MssqlConnectionPluginCApiRegisterWithRegistrar(
    FlutterDesktopPluginRegistrarRef registrar)
{
    // No method channel needed - this is an FFI plugin.
    // FreeTDS DLLs are loaded dynamically via DynamicLibrary.open()
}

File: macos/Classes/MssqlConnectionPlugin.swift
import Cocoa
import FlutterMacOS

// This is a dummy plugin class required by Flutter's plugin registration system.
// The actual FFI bindings are handled by Dart FFI directly loading the
// bundled FreeTDS dynamic libraries.
public class MssqlConnectionPlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    // No method channel needed - this is an FFI plugin.
    // FreeTDS symbols are loaded dynamically via DynamicLibrary.open()
  }
}
